pkg/services/tcp_proxy_test.go

```diff
@@ -196,3 +197,104 @@ func TestTCPProxyServiceOutbound(t *testing.T) {
 		})
 	}
 }
+
+func TestTCPProxyInboundCfgRun(t *testing.T) {
+	type testCoverageItem struct {
```

Comment:
We should add the specific error we expect to get. This way we know the specific case is getting the error we expect and not a different error.